src/Pages/Myprofile/Myprofile.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Tear from './Components/Tear';
+import Mytear from './Components/Mytear';
+import './Myprofile.scss';
+
+const NavCotent = [
+  {id: 0, name:'내 프로필'},
+  {id: 1, name:'맴버십혜택'},
+  {id: 2, name:'주문/배송조희'},
+  {id: 3, name:'내 쿠폰'},
+  {id: 4, name:'고객센터'},
+]
+
+const Information = [
+  {name: "내 라코스테 부티크", content: "자주 찾는 부티크를 추가해주세요", link: "+부티크 추가하기"},
+  {name: "주소록", content: "알려진 주소가 없습니다", link: "+새로운 주소 추가하기"},
+]
+
+class Myprofile extends Component {
+  constructor() {
+    super()
+    this.state = {
+      gradeList: [],
+      UserList: [],
+    }
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/data.json', {})
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          gradeList : data,
+        })
+      })
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/data.json')
      .then(response => response.json())
      .then(gradeList => this.setState({ gradeList }))
      })
```
위와 같이 수정할 수 있습니다.
- fetch 함수의 default method는 get입니다. 두번째 인자 생략해주셔도 됩니다.
- public에 접근할 때는 domain, port 생략해주셔도 됩니다.
- 객체의 키값과 value 자리의 변수명이 같을 경우 생략해줄 수 있습니다.